### DISCUSSION TITLE
Les jeux compatible

### DISCUSSION THREAD

**Discussion by Iliasouattara15** [skyline-emu/skyline#1878] [Game Requests]
Est-ce que cette émulateur peut jouer un autre jeu bien à part Sonic mania

**Comment by parknich082** [comment]
speak english dumbass

**Reply by Dekuzearou** [reply]
No need to be so agressive calm down smh my head

**Reply by Iliasouattara15** [reply]
Si tu comprends pas français c'est pas de ma faute idiot que tu es

**Comment by jin81790** [comment]
Naruto storm series specially 4 , kakarot and also pirate warriors 4 with DLC/ Update support that'd be really really great.well I'm obviously asking for a lot but I'm pretty sure everyone wants it too

**Comment by jin81790** [comment]
And like the citra mmj it'd be cool if we could use cheat but not too much just fast lvl up is good enough
